src/FormBuilder.php

```diff
@@ -823,6 +825,70 @@ protected function getControllerAction($options)
 	}
 
 	/**
+	 * Get the method for the route
+	 *
+	 * @param  array  $options
+	 * @return string
+	 */
+	protected function getHttpVerb( array $options )
+	{
+		$method = 'post';
+
+		// Look up named or controller routes, and get a method for them
+		if (isset($options['route']))
+		{
+			$method = $this->getVerbFromRouteAction($options['route']);
+		}
+		elseif (isset($options['action']))
+		{
+			$method = $this->getVerbFromControllerAction($options['action']);
+		}
+
+		// Override with explicitly defined method
+		return array_get($options, 'method', $method);
+	}
+
+	/**
+	 * Get the method from a named route
+	 *
+	 * @param  array|string $options
+	 * @return string
+	 */
+	protected function getVerbFromRouteAction($options)
+	{
+		if (is_array($options))
+		{
+			$name = $options[0];
+		}
```

Comment:
This if statement is immediately overwritten by the line #864 and therefore does nothing. Was that the intention?